docs/pages/enroll-resources/application-access/cloud-apis/aws-console-without-agent.mdx

```diff
@@ -0,0 +1,363 @@
+
+1. Write the following contents to a file called `roles-anywhere-integration.yaml`:
+  ```yaml
+  kind: integration
+  sub_kind: aws-ra
+  version: v1
+  metadata:
+    name: <Var name="Integration Name" />
+  spec:
+    aws_ra:
+      trust_anchor_arn: <Var name="Trust Anchor ARN" />
+      profile_sync_config:
+        enabled: true
+        profile_arn: <Var name="Profile ARN" />
+        role_arn: <Var name="Role ARN" />
+  ```
+
+1. Create the integration with the following command:
+  ```code
+  $ tctl create -f roles-anywhere-integration.yaml
+  ```
+
+Teleport will now start syncing AWS IAM Roles Anywhere Profiles as AWS application resources, every 5 minutes.
+
+## Step 2/4. Create an AWS IAM Roles Anywhere Profile and assign IAM Roles
+
+In this step, you will create an example Profile and Role so that you can test the integration.
+
+If you are already leveraging AWS IAM Roles Anywhere Profiles, you can skip this step.
+
+1. Navigate to [Create Role](https://console.aws.amazon.com/iam/home?#/roles/create) and create a new IAM Role called `ExampleReadOnlyAccess`.
+Trust policy must include the Roles Anywhere service principal:
+  ```json
+  {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Effect": "Allow",
+              "Principal": {
+                  "Service": "rolesanywhere.amazonaws.com"
+              },
+              "Action": [
+                  "sts:AssumeRole",
+                  "sts:SetSourceIdentity",
+                  "sts:TagSession"
+              ]
+          }
+      ]
+  }
+  ```
+
+Add the AWS-managed `ReadOnlyAccess` policy to the role.
+Copy the Role ARN <Var name="Read Only Role ARN" />.
+
+1. Now, navigate to [Create a Profile](https://console.aws.amazon.com/rolesanywhere/home/profiles/create) page and name it `ExampleReadOnlyAccess`.
+Add the role created in the last step (`ExampleReadOnlyAccess`) and create the Profile.
+
+Copy the Profile ARN <Var name="Read Only Profile ARN" />.
+
+1. You should now see the `ExampleReadOnlyAccess` profile listed in your Teleport Web UI in the Resources page.
+
+## Step 3/4. Set up Access
+In this section, you'll create a Teleport Role which will grant access to the `ExampleReadOnlyAccess` Profile and associated IAM Role.
+
+1. Create a file named `example-read-only-access-role.yaml` with the following contents:
+  ```yaml
+  kind: role
+  version: v8
+  metadata:
+    name: aws-ro-access
+  spec:
+    allow:
+      app_labels:
+        'teleport.dev/aws-roles-anywhere-profile-arn': '<Var name="Read Only Profile ARN" />'
+      aws_role_arns:
+      - <Var name="Read Only Role ARN" />
+  ```
+
+1. Create the role with the following command:
+  ```code
+  $ tctl create -f example-read-only-access-role.yaml
+  ```
+
+1. Assign the Role to the users you want to allow access. You can also assign it to an Access List or use it in conjunction with Access Requests.
```

Comment:
Nit: Link to corresponding access list and access request guides here. Otherwise reader has to go hunt for them.